napari/_vispy/layers/labels.py

```diff
@@ -236,6 +288,7 @@ def _on_colormap_change(self, event=None):
                 interpolation='nearest',
             )
             self.node.shared_program['LUT_shape'] = key_texture.shape
+            self.node.shared_program['color_count'] = len(color_dict) + 1
```

Comment:
Hm. I do not know what I should use here. Using `len(color_dict) + 1` will spedup stop searching for missing entries, but may cast some real big values to transparent if a collision happens.

Reply:
How so? Suppose all the labels are 1...n, then one more label m such that m lands on 1. This means that it will have to scan through 1-n, then land on n+1. So `len(color_dict)+1` will be sufficient to not skip that possibility, right? In what other scenario will it be too small?

Reply:
For keys above `2**23`. But I may be wrong.

Reply:
Well, this is handled by a uint32 (nice improvement!), so I don't think it will be an issue? (you will run out of texture shape before you run out of precision / overflow.)